dipy/denoise/tests/test_nlmeans.py

```diff
@@ -18,14 +18,14 @@ def test_nlmeans_padding():
 
 def test_nlmeans_static():
     S0 = 100 * np.ones((20, 20, 20), dtype='f8')
-    S0n = nlmeans(S0, sigma=1, rician=False)
+    S0n = nlmeans(S0, sigma=np.ones((20, 20, 20)), rician=False)
```

Comment:
The tests are defined on the inner nlmeans cython function, while the wrapper nlmeans sets everything for you. Because of this, I had to change the tests or they would just not work since hte inner mechanic is different.

The 3D/4D caller function still work as-is, which is what people are supposed to be calling.